packages/react-native-web/src/vendor/react-native/FlatList/index.js

```diff
@@ -8,6 +8,8 @@
  * @format
  */
 
+'use client';
```

Comment:
Can this move to [`exports/FlatList`](https://github.com/necolas/react-native-web/tree/master/packages/react-native-web/src/exports/FlatList)?